enterprise/backend/src/metabase_enterprise/serialization/v2/seed_entity_ids.clj

```diff
@@ -34,13 +34,14 @@
 (defn- make-table-name->model
   "Create a map of (lower-cased) application DB table name -> corresponding Toucan model."
   []
-  (into {} (for [model (descendants :toucan1/model)
-                 :when (models/model? model)
-                 :let  [table-name (some-> model t2/table-name name)]
-                 :when table-name
-                 ;; ignore any models defined in test namespaces.
-                 :when (not (str/includes? (namespace model) "test"))]
-             [table-name model])))
+  (into {}
+        (for [model (concat (descendants :toucan1/model) (descendants :metabase/models))
```

Comment:
see this for background on :metabase/models https://github.com/metabase/metabase/pull/29785#pullrequestreview-1375413845